bokeh/command/subcommands/serve.py

```diff
@@ -309,7 +309,9 @@
 from bokeh.server.tornado import DEFAULT_WEBSOCKET_MAX_MESSAGE_SIZE_BYTES
 from bokeh.settings import settings
 
-from os import getpid
+import tornado.autoreload as tar
```

Comment:
`from tornado.autoreload import watch` would would be more consistent with the rest of the codebase